packages/components/app/styles/components/side-nav/main.scss

```diff
@@ -66,56 +72,98 @@
 
 .hds-side-nav__menu-toggle-button {
   position: absolute;
+  top: -2px;
+  left: -1px;
```

Comment:
tip: abstract away this size in a variable (Sass or CSS) so it can be shared across different parts of this CSS implementation (they depend one on another)
<img width="213" alt="screenshot_3080" src="https://github.com/hashicorp/design-system/assets/686239/eb697eee-d8b1-4d9c-a261-0b78c5742423">

Reply:
(don't worry about the image, I am changing stuff to understand how you built the code to achieve the rounded tab)

Reply:
I refactor the styling of these elements using CSS variables and tokens so we can easily change the border (`width`, `color`, and `radius`) in future. If you'd like to have a play, here are the associated variables:
```css
  --hds-app-sidenav-border-width
  --hds-app-sidenav-border-color
  --hds-app-sidenav-border-radius
```